.github/workflows/fetch-ci-nightly-data.yaml

```
@@ -13,17 +15,20 @@ jobs:
       - name: Update dashboard data
         run: |
           # fetch ci nightly data as temporary file
-          node js/fetch-ci-nightly-data.js | tee tmp-data.js
+          node scripts/fetch-ci-nightly-data.js | tee tmp-data.json
           # switch to a branch specifically for holding latest data
           git config --global user.name "GH Actions Workflow"
           git config --global user.email "<gha@runner>"
           git fetch --all
           git checkout latest-dashboard-data
           # back out whatever data was there
           git reset HEAD~1
+          # pull in the latest changes
+          git pull
           # overwrite the old data
+          git rm -r data/
```

Let's revert to minimize the amount of changes, unless we have a strong reason to do this
```suggestion
          # overwrite the old data
```